packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -348,7 +348,7 @@ For more information, see the [Amazon Q documentation.](https://docs.aws.amazon.
 
         if (!cancelled) {
             message =
-                'The transformation job has been completed. If you want to start another transformation, choose **Start a new transformation.**'
+                'The transformation job is over. If you want to start another transformation, choose **Start a new transformation.**'
```

Comment:
We can follow up on exact wording later. Right now, this text is shown when a job fails (ex. network error when polling) and also at end of a successful job, so need to make it a bit more generic so as to not be misleading. Long-term fix, we will use the job status to customize the exact message shown.